Automatic Validation ended with:
> 2025-02-05T16:13:22.2658367Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.6.0 File: ScreenCloud.ScreenCloudPlayer.locale.zh-CN.yaml
 2025-02-05T16:13:22.2680883Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.6.0 File: ScreenCloud.ScreenCloudPlayer.locale.zh-CN.yaml

(Automated response - build 977.)

--- a/manifests/s/ScreenCloud/ScreenCloudPlayer/3.10.3/ScreenCloud.ScreenCloudPlayer.locale.zh-CN.yaml
+++ b/manifests/s/ScreenCloud/ScreenCloudPlayer/3.10.3/ScreenCloud.ScreenCloudPlayer.locale.zh-CN.yaml
@@ -1,5 +1,5 @@
 # Created with YamlCreate.ps1 Dumplings Mod
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
 
 PackageIdentifier: ScreenCloud.ScreenCloudPlayer
 PackageVersion: 3.10.3
@@ -23,4 +23,4 @@ Documentations:
 - DocumentLabel: 常见问题
   DocumentUrl: https://screencloud.com/faqs
 ManifestType: locale
-ManifestVersion: 1.6.0
+ManifestVersion: 1.9.0

